### ISSUE TITLE
[BITV] 9.1.1.1a/3.2 - Modal Dialog "Choose Target Folder" - The close button "X" does not have alternative text. (2)

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36715]
- [x] https://github.com/nextcloud/server/pull/36772
- [x] https://github.com/nextcloud/server/pull/37064
- [x] https://github.com/nextcloud/server/pull/37063

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v3-n2

</details>

- [x] 9.1.1.1a/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v10-n2
